scripts/requirements.txt

```diff
@@ -1,3 +1,7 @@
 GitPython==3.1.27
 pyyaml==5.4
-requests==2.27.1
\ No newline at end of file
+requests==2.27.1
+tqdm==4.63.0
+torch==1.12.1
+pandas
+mlflow

```

Comment:
We do not need these directories for this PR, since we do not have MLFlow support yet. Removing these dependencies for now. For future mlflow model support, we can discuss how to structure the files to reduce unnecessary increase in overall package size